MobiusExtras/Source/MobiusViewController.swift

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 Spotify AB.
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#if canImport(UIKit)
+
+import MobiusCore
+import UIKit
+
+public extension UIViewController {
+    /// Attach a `MobiusController` to this `UIViewController`.
+    /// The `connectable` you provide will be connected to the `MobiusController`.
+    /// This function returns a `Disposable` which can be disposed to stop the `MobiusController`. If this disposable
```

Comment:
I think it would be good to document best practices here. My thinking around that would be:

- Call `useMobius` in `viewDidLoad`.
- Dispose the `Disposable` if `didReceiveMemoryWarning` is called.
- The `Disposable` will be automatically disposed when the view controller is disposed.

Reply:
`useMobius` can be called in `init` _or_ `viewDidLoad`, but I would tend to prefer `init`. The `MobusController` should not be torn down in `didReceiveMemoryWarning`; maintaining the state you need to recreate it correctly would unnecessarily duplicate state from the `MobiusController` and the model and increase overall memory usage.